outlook/src/classes/Address.ts

```diff
@@ -14,37 +14,31 @@ class Address {
     }
 
     getLines(): string[] {
-        let firstLine = [this.number || "", this.street || ""].join(" ").trim();
-        let secondLine = [this.city || "", this.zip || ""].join(" ").trim();
+        const firstLine = [this.number || '', this.street || ''].join(' ').trim();
+        const secondLine = [this.city || '', this.zip || ''].join(' ').trim();
 
-        let lines = [];
-        if (firstLine)
-            lines.push(firstLine)
-        if (secondLine)
-            lines.push(secondLine)
-        if (this.country)
-            lines.push(this.country)
+        const lines = [];
+        if (firstLine && firstLine.length) lines.push(firstLine);
```

Comment:
Is it standard to not use parenthesizes ? Seems you are not shocked so fine for me :) . Just noting it once.